packages/admin/admin/src/form/FinalFormContextProvider.tsx

```diff
@@ -5,12 +5,14 @@ export interface FinalFormContext {
     shouldScrollToField: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
     shouldShowFieldError: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
     shouldShowFieldWarning: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
+    shouldShowFieldHelperText: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
 }
 
 const defaultFinalFormContext: FinalFormContext = {
     shouldScrollToField: () => false,
     shouldShowFieldError: ({ fieldMeta }) => !!fieldMeta?.touched,
     shouldShowFieldWarning: ({ fieldMeta }) => !!fieldMeta?.touched,
+    shouldShowFieldHelperText: ({ fieldMeta }) => !!fieldMeta?.touched,
```

Comment:
I don't think this logic is required or wanted. A help text should always be shown, independent of if this field was touched. Only if a warning or error is shown, the help text is not sown. (it indirectly still depends on touched)

(at least that would make the most sense to me)

Reply:
👍  I will remove it